modules/ossm-validating-operators.adoc

```diff
@@ -0,0 +1,65 @@
+// Module included in the following assemblies:
+// * service_mesh/v2x/-ossm-troubleshooting-istio.adoc
+
+[id="ossm-validating-operators_{context}"]
+= Validating Operator installation
+
+//The Operator installation steps include verifying the Operator status in the OpenShift console.
+
+When you install the {ProductName} Operators, OpenShift automatically creates the following objects as part of a successful Operator installation:
+
+* `ConfigMaps`
+* `CustomResourceDefinitions`
+* `Deployments`
+* `Pods`
+* `ReplicaSets`
+* `Roles`
+* `RoleBindings`
+* `Secrets`
+* `ServiceAccounts`
+* `Services`
```

Comment:
not sure if I left this comment before, but since `ThisStyle` indicates a literal name of an object, I'd use the general way or referring to them instead, since some items in this list are pluralized. E.g. `ConfigMap` resource or `ServiceAccount` object is a literal thing, but for general plural use I believe it should be "config maps" or "service accounts"

https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#general-references